onfido-java/pom.xml

```diff
@@ -4,7 +4,7 @@
 
     <groupId>com.onfido</groupId>
     <artifactId>onfido-api-java</artifactId>
-    <version>2.8.0</version>
```

Comment:
@dvacca shouldn't this be changed by some kind of a release job?

Reply:
@lotr that's fine like this, we just bump the version before releasing